writer_test.go

```diff
@@ -313,6 +328,43 @@ func TestSetMapForMediaPlaylist(t *testing.T) {
 	if e != nil {
 		t.Errorf("Set map to a media playlist failed: %s", e)
 	}
+
+	expected := "EXT-X-MAP:URI=\"https://example.com\",BYTERANGE=1024000@1048576\n#EXTINF:5.000,\ntest01.ts"
```

Comment:
thanks, have updated test expectations to string literals where possible.  hopefully good to merge now.